### PR TITLE
use original logo

### PR DESCRIPTION
**Pull request by ry**
Screen Shot 2022-04-01 at 6 38 46 PM

### CODE REVIEW

**Comment by littledivy** [comment]
Screenshot 2022-04-02 at 11 19 16 AM

Very blurry for me

File: static/logo.svg [hunk 1]
```diff
@@ -1 +1,10 @@
...
...R85CI1LIBSVRk+qnoSP9qSA0gAAEIQAACEOiCwLL+hREYOr21Or1cpz9bm/VBFtT/3IWwUUgIQAACEIAABOwSkA7JNE3sQMboC+tDsIG/SFTsaj01gwAEIAABCJghQALT391wRjBos7s2QKJiRsKpCAQgAAEIQGA8AiQwdIbvdob5vV4bIlEZT9OpMQQgAAEIQMA8gTmB+SRPt6YjqrcjStvQNns2MPuv7PrFGhXzqk0FIQABCEAAAoMT8J4D8xs7kdFJ3usk87/m9rE88PEDz1EZXLipPgQgAAEIQGB0ArIF6ryQn1EYFnKzmL+xDcyjKbLjF1sTjy7O1B8CEIAABCAAgTiB+WGWH51zMgpDBxYG2EAFG5j97SMPe4zrEkchAAEIQAACEIDALgFGYUjcSF7z24C3NoXRlF0F4p8QgAAEIAABCEDgJIF5LcwvTCXL34klMbDN1Jvy9QtrU04KD1+HAAQgAAEIQAACdwiQxNjuaJNIXW9fkpQ7ysJvIQABCEAAAhCAQEECSxLjnPt17rSxHqLCegiSi+vJRS52JCkFhYVTQwACEIAABCAAgZIE5iRGdiaT58N8YXvl9p3rXJ30wc8j2xCLPcvzUv7JdK+SKsK5IQABCEAAAhCAQAMC8lC8x+PxgdEYEpheEh9vFEWelcJDHRvoBpeEAAQgAAEIQAACzQnMO5TJaAwPu2Q6WesphTKKInb4PorS3DkoAAQgAAEIQAACEICATgLyrAo/kXHOfe/lrjzl7GcESezKT1DE6HR6BKWCAAQgAAEIQAACEOiGwJzIfJy3XJY1Ba3vynP9TtpgXoPyeZomeYAjz0bpxuspKAQgAAEIQAACEDBAYF4jI8+OeV/sz6jM2IncPHqyLJKXZ6KwBsWAn1MFCEAAAhCAAAQgYJJAkMzIFsyMzHQyKpIygibtKZs5zGtPSE5MejGVggAEIAABCEAAAoMS8LZhlmlmy+jM15SOMt+pO4oju3Z5WwvLzl1sLzyo31JtCEAAAhCAAAQgAIGZwDw6Ix3jD5LQeEmN3NWXkZo3Epcsicv7c07m0ZL3qVzziMmSmDClC6+EAAQgAAEIQAACEIBADgJekiOJzvufLPReEp7wdemkJ7w22S1tWQ+SUL5nohHWcVnovvCYX0lCchgc54AABCDQiMD/B+aGL+yADhfgAAAAAElFTkSuQmCC"/>
```
Comment (kt3k) [inline static/logo.svg:8]: Looks like this is an embedded png

Maybe we should use [this](https://raw.githubusercontent.com/denolib/high-res-deno-logo/master/deno_hr_circle.svg) instead?

Comment (ry) [reply static/logo.svg]: Fixed

**Comment by roj1512** [comment]
Maybe it should be a little larger? It is hard on the eyes.

**Review by kt3k — APPROVED** [review]

**Comment by ry** [comment]
Closing without merge - looks kinda weird

**Pull request closed without merging**